cms/src/components/content/EditContentParentValidation.vue

```diff
@@ -142,6 +143,21 @@ watch(
 <template>
     <div class="rounded-md bg-zinc-100 p-3 shadow-inner">
         <div class="flex flex-col gap-2">
+            <div v-if="!parentIsValid" class="rounded-md bg-zinc-50 p-4 shadow">
+                <span class="text-sm">Errors were found in your {{ parent?.type }}'s settings</span>
+                <div class="flex flex-col gap-0.5">
+                    <div
+                        v-for="validation in parentValidations.filter((v) => !v.isValid)"
+                        :key="validation.id"
+                        class="flex items-center gap-1"
+                    >
+                        <p class="flex items-center gap-1">
+                            <XCircleIcon class="h-4 w-4 text-red-400" />
+                            <span class="text-sm">{{ validation.message }}</span>
```

Comment:
The text here is bigger (and perhaps darker?) than of the ContentDto validation messages.